### ISSUE TITLE
Formidable/MapPress is not sending the API key with Google maps requests in forms

### ISSUE THREAD

**Issue by Photosynthesis** [Foundation-For-Intentional-Community/Wordpress-Theme#33]
The input forms for adding or editing a community are showing a Google error: 

![image](https://user-images.githubusercontent.com/4992900/95000629-73624800-0577-11eb-884a-d8f1c40ffffa.png)

Looking at the console, this appears to be because the Google API key is not being sent with the request to the API.

**Comment by Photosynthesis** [comment]
The MapPress admin page shows a demo map, and source for that page shows that the API key is being sent. This makes it seem like the issue is not with MapPress.

**Comment by Photosynthesis** [comment]
The main directory page is also using the API key.

**Comment by Photosynthesis** [comment]
MapPress is not used for the inputs, which are instead generated by a small and old plugin called Frm Google Maps. This plugin did not have support for the Google API key. I have added it, but this is only a temporary solution. The plugin seems to be unmaintained and not available on the web, but if it was updated and the update was applied it would remove the modification. Longer term this will be moved into directory plugin code as we transition away from Formidable.
